src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1272,6 +1273,7 @@ namespace UnitarySystems {
                 }
             }
         }
+        return false; // these later errors do not cause a fatal at this time
```

Comment:
Since function was changed to bool, bottom of checkNodeSetPoint returns false and acts the same way as it does now with no fatal on temp/humrat set point errors. We can change this if we think humidity control set point is an issue.

Reply:
Local regressions and unit tests all pass so it seems that adding the fatal error to existing set point warnings would be an improvement.